tests/c_languages/CPPCleanBearTest.py

```diff
@@ -17,6 +18,41 @@
 """
 
 
+test_include_paths_file = """
+#include "foo.h"
+#include "bar.h"
+
+int main() {
+    foo();
+    bar();
+    return 0;
+}
+"""
+
+
+def get_testdirectory_name(test_directory):
+    return os.path.join(os.path.dirname(__file__),
+                        'test_files',
+                        test_directory)
+
+
 CPPCleanBearTest = verify_local_bear(CPPCleanBear,
                                      valid_files=(good_file,),
-                                     invalid_files=(bad_file,))
+                                     invalid_files=(bad_file,
+                                                    test_include_paths_file,),
+                                     tempfile_kwargs={'suffix': '.cpp'})
+
+CPPCleanBearValidIncludeMultiplePathTest = verify_local_bear(
+    CPPCleanBear,
+    valid_files=(test_include_paths_file,),
+    invalid_files=(),
+    settings={'include_paths': get_testdirectory_name('headers1') + ',' +
+              get_testdirectory_name('headers2')},
+    tempfile_kwargs={'suffix': '.cpp'})
+
+CPPCleanBearInvalidIncludeSinglePathTest = verify_local_bear(
+    CPPCleanBear,
+    valid_files=(),
+    invalid_files=(test_include_paths_file,),
+    settings={'include_paths': get_testdirectory_name('headers1')},
```

Comment:
ah I just remember another PR, where this has lead to confusion: Please add this to your test:
```
from coala_utils.string_processing import escape
```
and use `escape` on each of those directory names:
```
```
coala preprocesses escape characters. Paths in Windows use backslashes, which yields to test fails because files can't be found (`C:\xyz\abc` gets `C:xyzabc`). To prevent that, we have to escape the backslashes, and `escape` can do that for us:
```
escape(get_testdirectory_name(...), '\\')
```
We have to do something about it in the future so we don't get sudden test fails on Windows due to this^^

Reply:
@Makman2 Ok I can do that, just one thing is this like the same issue xmlbeartest is facing? I remember while looking into it there was problem with tests where directories were being used, just something i remembered could be totally wrong :P

Reply:
yes it was one of the issues :+1: